content/community_calls/2026-02-18 Notes.en.md

```
@@ -0,0 +1,47 @@
+---
+title: "2026-02-18 Notes"
+weight: 968
+type: "post"
+date: 2026-02-18
+---
+
+### Recording
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)